src/App.js

```diff
@@ -1,25 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox'
+import AddFoodForm from './components/AddFoodForm'
+import Search from './components/Search'
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+
+
+
+    const createFood = (newDishes) => {
+      const updatedDishes = [newDishes, ...dishes]
+      setDishes(updatedDishes)
```

Comment:
Every time you create a food you need to update the dishes like you are doing here.
But now since you have another useState that will be the result of the search you need to update it as well. 
So under the ` setDishes(updatedDishes)` add this line `setShowFoods(updatedDishes)`